assets/js/sqoot.js

```diff
@@ -1,254 +0,0 @@
-var mainDealDisplay = [];
-var items = [], info= [];
-var userlat, userlng, userSearch, userLocation, length, address;
-
-
-$("form").on("submit", function(event) {
-  event.preventDefault();
-
-  $('#deal-table-panel').remove();
-
-  //Get items from input form and assign them to global variables
-  userSearch = $(".user-search-term").val().trim();
-  userLocation = $(".user-location").val().trim();
-
-  //Console log the two search terms to make sure they are being retrieved from the UI
-  console.log(userSearch);
-  console.log(userLocation);
-
-  var webResults = false;
-  var results = 5;
-
-  //Set the UI input fields back to blank for the user
-  $(".user-search-term").val('');
-  $(".user-location").val('');
-
-  //Start building the elements inside the deal-table-row, including panel, and table that will contain the deal data
-  var boostrapTablePanel     = $('<div class="panel panel-default" id="deal-table-panel">'),
-      boostrapTablePanelBody = $('<div class="panel-body" id="deal-table-panel-body">');
-  var tableWhole = $('<table class="table table-striped" id="main-deal-table"><tbody></tbody></table>')
-  var tableHeaderRow = $('<tr id="table-header-row">')
-
-  //Build the table header
-  var tableHeader0 = $('<th>').text('Favorite');
-  var tableHeader1 = $('<th>').text('Business Name');
-  var tableHeader2 = $('<th>').text('Deal');
-  var tableHeader3 = $('<th class="centerText">').text('Deal Price');
-  var tableHeader4 = $('<th class="centerText">').text('Original Value');
-  var tableHeader5 = $('<th class="centerText">').text('Discount');
-
-  //Append the table panel to the deal-table-row that is on the index page to hold the table
-  $('#deal-table-row').append(boostrapTablePanel);
-  //Append the table panel body to the panel that was just created and appended
-  $('#deal-table-panel').append(boostrapTablePanelBody);
-  //Append the table to the panel body that was just created and appended
-  $('#deal-table-panel-body').append(tableWhole);
-
-  //Fill out the header row with the relevant headers, then append the header to the table
-  tableHeaderRow.append(tableHeader0);
-  tableHeaderRow.append(tableHeader1);
-  tableHeaderRow.append(tableHeader2);
-  tableHeaderRow.append(tableHeader3);
-  tableHeaderRow.append(tableHeader4);
-  tableHeaderRow.append(tableHeader5);
```

Comment:
Although this works, it may be useful to know that jQuery's .append can take multiple arguments.
The following works:
```
tableHeaderRow.append(tableHeader0, tableHeader1, tableHeader2, tableHeader3, tableHeader4, tableHeader5);

var tableHeaders = [tableHeader0, tableHeader1, tableHeader2, tableHeader3, tableHeader4, tableHeader5];
tableHeaderRow.append(tableHeaders);